src/test/java/org/killbill/billing/plugin/vertex/VertexInvoicePluginApiITest.java

```diff
@@ -0,0 +1,253 @@
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.killbill.billing.plugin.vertex;
+
+import java.math.BigDecimal;
+import java.util.Collection;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.UUID;
+
+import org.killbill.billing.account.api.Account;
+import org.killbill.billing.catalog.api.CatalogApiException;
+import org.killbill.billing.catalog.api.CatalogUserApi;
+import org.killbill.billing.catalog.api.Currency;
+import org.killbill.billing.catalog.api.StaticCatalog;
+import org.killbill.billing.invoice.api.Invoice;
+import org.killbill.billing.invoice.api.InvoiceItem;
+import org.killbill.billing.invoice.api.InvoiceItemType;
+import org.killbill.billing.invoice.api.InvoiceUserApi;
+import org.killbill.billing.osgi.libs.killbill.OSGIConfigPropertiesService;
+import org.killbill.billing.osgi.libs.killbill.OSGIKillbillAPI;
+import org.killbill.billing.payment.api.PluginProperty;
+import org.killbill.billing.plugin.TestUtils;
+import org.killbill.billing.plugin.api.PluginCallContext;
+import org.killbill.billing.plugin.vertex.base.VertexRemoteTestBase;
+import org.killbill.billing.util.api.CustomFieldUserApi;
+import org.killbill.billing.util.callcontext.CallContext;
+import org.killbill.clock.Clock;
+import org.killbill.clock.DefaultClock;
+import org.mockito.Mockito;
+import org.osgi.framework.BundleContext;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+// Note: the test assumes all California authorities are set up to collect sales and use tax (270+)
+public class VertexInvoicePluginApiITest extends VertexRemoteTestBase {
+
+    private OSGIKillbillAPI osgiKillbillAPI;
+    private Collection<PluginProperty> pluginProperties;
+    private VertexTaxCalculator vertexTaxCalculator;
+    private VertexInvoicePluginApi vertexInvoicePluginApi;
+    private Account account;
+    private CallContext callContext;
+
+    @BeforeMethod(groups = "integration")
+    public void setUp() throws Exception {
+        final Clock clock = new DefaultClock();
+
+        pluginProperties = new LinkedList<>();
+        pluginProperties.add(new PluginProperty(VertexConfigProperties.VERTEX_OSERIES_COMPANY_DIVISION_PROPERTY, "328", false));
+        pluginProperties.add(new PluginProperty(VertexConfigProperties.VERTEX_OSERIES_COMPANY_NAME_PROPERTY, "Kill Bill Parent", false));
+
+        // California Nexus must be enabled in your account for the test to pass
+        // As of July 2021, tax rates are:
+        //   CA STATE TAX: 0.06
+        //   CA COUNTY TAX: 0.0025
+        //   CA SPECIAL TAX (SAN FRANCISCO CO LOCAL TAX SL): 0.01
+        //   CA SPECIAL TAX (SAN FRANCISCO COUNTY DISTRICT TAX SP): 0.01375
```

Comment:
is this relevant?

Reply:
Good catch, checked CA Sales tax and it is 7.25 as of 2023 - decided to remove this comments to not to maintain it